app/models/employee.rb

```diff
@@ -3,4 +3,7 @@ class Employee < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :employee
+  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :employee
```

Comment:
hej `inverse_of` podpowiada railsom co jest odwrotnością relacji
przykład: `Kudo has many given_kudos` -> odwrotnością tego jest `Kudo belongs to giver`
a więc w tym przypadku powinieneś wstawić tutaj `:giver` a nie employee, oraz analogicznie dla receivera